pkgs/misc/cliscord/default.nix

```diff
@@ -1,21 +1,29 @@
-{ lib, stdenv, rustPlatform, openssl, pkg-config, fetchFromGitHub, Security }:
+{ lib
+, stdenv
+, rustPlatform
+, openssl
+, pkg-config
+, fetchFromGitHub
+, fetchpatch
+, Security
+}:
 
 rustPlatform.buildRustPackage rec {
   pname = "cliscord";
-  version = "unstable-2020-12-08";
+  version = "0.0.2";
```

Comment:
```suggestion
  version = "unstable-2022-10-07";
```

Reply:
Done! This also helped me realize that I think I forgot to update the hash for the fixed-output vendor derivation, so that should be fixed also.